rosidl_runtime_c/include/rosidl_runtime_c/string_functions.h

```diff
@@ -26,44 +26,118 @@ extern "C"
 #endif
 
 /// Initialize a rosidl_runtime_c__String structure.
-/* The contents of rosidl_runtime_c__String are initialized to a single null character ('\0').
+/*
+ * The contents of rosidl_runtime_c__String are initialized to a single null character ('\0').
  * The string initially has size 0 and capacity 1.
  * Size represents the size of the contents of the string, while capacity represents the overall
  * storage of the string (counting the null terminator).
  * All strings must be null-terminated.
+ * The rosidl_runtime_c__String structure should be deallocated using the given function
+ * rosidl_runtime_c__String__fini() when it is no longer needed.
+ *
+ * \param[inout] str a pointer to a rosidl_runtime_c__String structure
```

Comment:
From what I can find the proper syntax is `[in,out]`, but I've also seen `[in|out]`. But I think `[inout]` is incorrect. Fix here and below

Reply:
I'm using the same style as other ROS 2 packages. For example: [rcl](https://github.com/ros2/rcl/blob/master/rcl/include/rcl/client.h#L139)